community/mtd-utils/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=mtd-utils
+pkgver=2.1.0
+pkgrel=0
+pkgdesc="Utilities for handling MTD devices, and for dealing with FTL, NFTL JFFS2, etc."
+url="http://www.linux-mtd.infradead.org/"
+arch="all"
+license="GPL-2.0"
+makedepends="
+	acl-dev
+	linux-headers
+	lzo-dev
+	openssl-dev
+	util-linux-dev
+	zlib-dev
+"
+subpackages="${pkgname}-doc"
```

Comment:
```suggestion
subpackages="$pkgname-doc"
```

Reply:
I know this is the 'style' used throughout a lot of APKBUILD's. But it's very poor style and is heavily frowned uppon :) so lets atleast make it proper where we are touching things anyway.

Proper shell variable usage is:
Use {} where possible, to clearly indicate the start/end of your variable; quote all strings.

Reply:
gpl-2 is not a SPDX license - https://spdx.org/licenses/

Reply:
I am all for "better styling", when the new org. structure is settled there should be plenty opportunity to make a working group which goal is to make/improve the style and improve the APKBUILD format.

Reply:
> gpl-2 is not a SPDX license identifier - https://spdx.org/licenses/
> 
> https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license

Yes, I understand what you mean now. Actually, no you are wrong; but you are right.
the -only and -later monikers are for SDPX v3 (or some version) whereas currently we use the v2 (or was it v1?) SPDX license identifier.

However, I completely agree with you in that we should use the proper identifier. I was not aware that there was the -only variant, I thought GPL-v2 IS the -only variant. My bad.